pkg/controller/volume/attachdetach/cache/actual_state_of_world.go

```diff
@@ -650,6 +653,25 @@ func (asw *actualStateOfWorld) GetNodesToUpdateStatusFor() map[types.NodeName]no
 	return asw.nodesToUpdateStatusFor
 }
 
+func (asw *actualStateOfWorld) GetVolumesInUpdateStatusFor() map[types.NodeName][]v1.UniqueVolumeName {
```

Comment:
This is *only* used in tests right? Can we add a comment for that? Also - I think this could be renamed to `GetVolumesInNodeStatus` because unlike `GetVolumesToReportAttached` this function returns all volumes which are in node's status.

Reply:
Thanks for your review. After myself review, I find **"GetVolumesToReportAttached"** function is ok for my test. My test case is to verify the volumes count in  **nodesToUpdateStatusFor**  is right. So **GetVolumesToReportAttached** is enough. And I remove my useless function.